iOS/ViewController.m

```diff
@@ -356,12 +356,10 @@ - (void)setARMode:(ARMode)mode {
     self.arEnabled = mode != ARModeDisabled;
     self.arMode = mode;
 
-    if(!wasEnabled && self.arEnabled) {
-        [self forceResetView];
-    }
-
-    if(wasEnabled && !self.arEnabled) {
+    if(wasEnabled ^ self.arEnabled) {
```

Comment:
Had to look up what `wasEnabled ^ self.arEnabled` was :)

Reply:
I just realized that in this case != is actually does the same thing, some pattern matching part of my brain went "oh, that's an exclusive or", which is true, but not super useful here. I'll change it to be a bit less opaque.